webcompat/static/css/development/components/report-form.css

```diff
@@ -15,6 +15,19 @@
     text-align: center;
     justify-content: center;
     flex-wrap: wrap;
+    margin-top: 1em;
+  }
+
+  .wc-ReportForm .wc-UploadForm {
+    border: dashed 1px rgb(204, 204, 204);
+    border-radius: 20px;
+  }
+
+  .wc-ReportForm .r-Grid-cell {
+    margin-bottom: 1em;
+  }
+  .wc-ReportForm div.r-minM--1of2:nth-child(2) .wc-Form-group {
```

Comment:
here definitely agree with you. But instead of another wrapper (aka div or span if I understood you), I would prefer a class attribute.